tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -118,8 +117,8 @@
                     "2vcpu_512mb.json": {"target": 60},
                 },
                 "async": {
-                    "2vcpu_256mb.json": {"target": 190},
```

Comment:
This change seems unrelated to the MAC address? Or does the snapshot latency performance change with this commit?

Reply:
I am not sure why this is in one of the commits, this change is not shown in the `file changes`